package.json

```diff
@@ -3,7 +3,8 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "node ./bin/www",
+    "start": "npm install && node ./bin/www",
+    "start-worker": "npm install && node ./bin/worker",
```

Comment:
Remove the npm installs. I remove it from the package.json in #42 it causes issues because both of the servers try to run it at the same time.